### PR TITLE
Chore: Move from master to main

### PR DESCRIPTION
**Pull request by molant**
Part of #3809 
All the other repos have been renamed but this one is a bit trickier so doing a PR in case someone catches something I missed.

Once this is merged the only remaining task would be to change the default branch and delete master.
If other PRs are merged in master before this PR should be rebased.

### CODE REVIEW

**Comment by molant** [comment]
Windows timedout while installing dependencies but the others pass with no pbms.

@sarvaje can you take a look?

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
-# Scope the validations to master and PRs to master
+# Scope the validations to main and PRs to main
 trigger:
-  - master
+    - main
```
Comment (sarvaje) [inline azure-pipelines.yml:3]: any reason for the extra spaces?

Comment (molant) [reply azure-pipelines.yml]: an autoformatter when saving 😠
Should be fixed now

**Comment by sarvaje** [comment]
@molant the timeout issue looks like it was a temporary issue, the build is passing now.

**Pull request merged**